server/events.go

```diff
@@ -569,14 +570,12 @@ func (s *Server) initEventTracking() {
 	}
 	s.sys.inboxPre = subject
 	// This is for remote updates for connection accounting.
-	subject = fmt.Sprintf(accConnsEventSubj, "*")
-	if _, err := s.sysSubscribe(subject, s.remoteConnsUpdate); err != nil {
-		s.Errorf("Error setting up internal tracking: %v", err)
-	}
-	// This will be for responses for account info that we send out.
-	subject = fmt.Sprintf(connsRespSubj, s.info.ID)
-	if _, err := s.sysSubscribe(subject, s.remoteConnsUpdate); err != nil {
-		s.Errorf("Error setting up internal tracking: %v", err)
+
+	for _, subj := range []string{accConnsEventSubjOld, accConnsEventSubjNew, connsRespSubj} {
+		subject = fmt.Sprintf(subj, "*")
```

Comment:
You are changing the subscription subject here for `connsRespSub`. Used to be on `s.info.ID` but you all 3 are on `*`. I am not sure this is right to have this server listen to all responses, should be only for itself.

Reply:
What about this concern?

Reply:
totally missed that one, thanks for catching that